ush/python/pygfs/task/atm_analysis.py

```diff
@@ -73,46 +70,42 @@ def __init__(self, config: Dict[str, Any], yaml_name: Optional[str] = None):
         # Extend task_config with local_dict
         self.task_config = AttrDict(**self.task_config, **local_dict)
 
-        # Create JEDI object
-        self.jedi = Jedi(self.task_config, yaml_name)
-
-    @logit(logger)
-    def initialize_jedi(self):
-        """Initialize JEDI application
-
-        This method will initialize a JEDI application used in the global atm analysis.
-        This includes:
-        - generating and saving JEDI YAML config
-        - linking the JEDI executable
-
-        Parameters
-        ----------
-        None
-
-        Returns
-        ----------
-        None
-        """
+        # Create dictionary of JEDI objects
+        self.jedi = AttrDict()
```

Comment:
I think we might want a better name for this object, but don't have a good suggestion. `self.jedi_tasks` maybe? just some way to make it clearer that it is a dictionary of Jedi class objects

Reply:
@CoryMartin-NOAA Good call, will change